doc/howto/network_acls.md

```diff
@@ -359,14 +361,245 @@ lxc network acl show-log <ACL_name>
 (network-acls-edit)=
 ## Edit an ACL
 
-Use the following command to edit an ACL:
+`````{tabs}
+````{group-tab} CLI
+
+Use the CLI to:
+
+- {ref}`network-acls-edit-cli-rename`
+- {ref}`network-acls-edit-cli-properties`
+
+(network-acls-edit-cli-rename)=
+### Rename an ACL via CLI
+
+Requirements:
+
+- You can only rename an ACL that is not currently assigned to a NIC or network. See the {ref}`network-acls-assign` section for more information.
+- The new name must meet the naming requirements listed in {ref}`network-acls-create`.
+
+To rename an ACL, query the following endpoint:
+
+```bash
+lxc network acl rename <old ACL name> <new ACL name>
+```
+
+
+(network-acls-edit-cli-properties)=
+### Edit other properties via CLI
+
+Run:
 
 ```bash
 lxc network acl edit <ACL_name>
 ```
 
-This command opens the ACL in YAML format for editing.
-You can edit both the ACL configuration and the rules.
+This command opens the ACL in YAML format for editing. You can edit any part of the configuration _except_ for the ACL name.
+
+````
+
+````{group-tab} API
+
+Use the API to:
+
+- {ref}`network-acls-edit-api-rename`
+- {ref}`network-acls-edit-api-properties`
+- {ref}`network-acls-edit-api-config`
+
+(network-acls-edit-api-rename)=
+### Rename an ACL via API
+
+Requirements:
+
+- You can only rename an ACL that is not currently assigned to a NIC or network. See the {ref}`network-acls-assign` section for more information.
+- The new name must meet the naming requirements listed in {ref}`network-acls-create`.
+
+To rename an ACL, query the following endpoint:
+
+```bash
+lxc query --request POST /1.0/network-acls/{name} --data '{
```

Comment:
We should use consistent placeholder names, so rather than `name` here, suggest `ACL_NAME` or `acl_name`, which ever we also use in the CLI examples.